bin/parse_viral_pred.py

```diff
@@ -372,14 +387,17 @@ def merge_annotations(pprmeta, finder, sorter, sorter2, assembly, vs_cutoff):
                     )
                 elif seq_record.id in pprmeta_lc and seq_record.id in finder_lowestc:
                     lc_predictions_contigs.append(seq_record)
-
+                    
+    print(f"Number of skipped contigs {count_skipped_contigs}")
```

Comment:
We should collect the names of the contigs that fail and report them, otherwise contigs may be discarded and users may not understand why. I would print the names to a log file (sanity_check_failed_contigs.txt or similar)